en/peripherals/rtk_gps_cuav_c-rtk.md

```diff
@@ -0,0 +1,43 @@
+# CUAV C-RTK
+
+The [CUAV C-RTK GPS receiver](http://doc.cuav.net/gps/c-rtk/en/) is a RTK module for mass market launched by CUAV. The complete RTK system consists of at least two C-RTK modules \(one for the base station and the others for the aircraft\). Using RTK, PX4 can get its position with centimetre-level accuracy, which is much more accurate than can be provided by a normal GPS.
+
+![C-RTK](../../images/rtk_c-rtk.jpg)
+
+## Purchase
+
+* [cuav taobao](https://item.taobao.com/item.htm?id=565380634341&spm=2014.21600712.0.0)
+* [cuav aliexpress](https://www.aliexpress.com/store/product/CUAV-NEW-Flight-Controller-GPS-C-RTK-differential-positioning-navigation-module-GPS-for-PIX4-Pixhawk-pixhack/3257035_32853894248.html?spm=2114.12010608.0.0.75592fadQKPPEn)
+
+## Configuration
+
+RTK setup and use on PX4 via _QGroundControl_ is largely plug and play \(see [RTK GPS](../advanced_features/rtk-gps.md) for more information\).
+
+## Wiring and Connections
+
+C-RTK GPS with 6-pin connector and 4-pin connector, 6-pin connector can be plugged directly into [Pixhack v3](http://doc.cuav.net/gps/c-rtk/en/Instructions.html) GPS UART port, 4-pin connector can not be connected or access pixhack v3 ser4/5 port
```

Comment:
> 4-pin connector can not be connected or access pixhack v3 ser4/5 port

So what is the 4 pin connector for - and what platforms can it be connected to? If it can be connected to anything probably worth listing that and doing pinout.

Reply:
So basically I think you are saying that you connect the 6 pin connector (only) for Pixhack, Pixhawk3Pro and Pixracer. For Pixhack you can connect directly, for the Pixhawk3 Pro and Pixracer you need to modify the cable. Is that correct?

Please confirm you do not need the 4 pin connector for those other boards.

Reply:
The 4pin connector can be connected to the uart4 of the px4 flight controller as gps2, but the px4 firmware has not found a way to enable the second gps.